PDSOutputer.cc

```diff
@@ -133,7 +141,11 @@ void PDSOutputer::writeFileHeader(std::vector<SerializerWrapper> const& iSeriali
   
   {
     //The file type identifier
-    const uint32_t id = 3141592*256+1;
+    uint32_t comp = 0;
+    if(serialization_ == Serialization::kRootUnrolled) {
+      comp = 1;
+    }
+    const uint32_t id = 3141592*256+1 + comp;
```

Comment:
I'm not thrilled with reusing the _file type identifier_ to also store the serialization type. Given that PDS is just a test case, and I want to be able to read older PDS files, I figured it is OK.